eng/common/scripts/Verify-Links.ps1

```diff
@@ -87,7 +87,7 @@ function ResolveUri ([System.Uri]$referralUri, [string]$link)
     return $null
   }
 
-  if ($null -ne $ignoreLinks -and $ignoreLinks.Contains($link)) {
+  if ($null -ne $script:ignoreLinks -and $script:ignoreLinks.Contains($link)) {
```

Comment:
Is this necessary? I wouldn't expect it to be.

Reply:
let me try removing it and test